packages/react-chat/.storybook/preview-head.html

```diff
@@ -7,4 +7,5 @@
 
 <script>
   window.global = window;
-</script>
+  window.__USE_SHADOW_ROOT__ = false;
```

Comment:
this var value is storybook-specific and will not affect the value of the global var. 
it's either this assignment here, or, if we don't want to mess with  `__USE_SHADOW_ROOT__` at all,  a check for [IS_STORYBOOK  ](https://storybook.js.org/docs/faq#how-can-my-code-detect-if-it-is-running-in-storybook)(SB global var)